src/services/read-from-disk.service.js

```diff
@@ -206,14 +211,24 @@ export function loadAllProjectDependencies(
   return loadPackageJson(projectPath).then(
     packageJson =>
       new Promise((resolve, reject) => {
-        const dependencyNames = Object.keys(packageJson.dependencies);
+        // Check for existence of both dependencies and devDependencies
+        // We can reasonably assume all projects have dependencies
+        // but some may not have devDependencies
+        const deps = Object.keys(packageJson.dependencies);
+        const devDeps = packageJson.devDependencies
+          ? Object.keys(packageJson.devDependencies)
+          : [];
+
+        // If devDeps exists, add to dependencyNames, otherwise don't
+        const dependencyNames = devDeps ? [...deps, ...devDeps] : [...deps];
```

Comment:
Nit: I don't think the ternary is necessary :) `devDeps` is guaranteed to be an array at this point. You can spread an empty array, it has no effect. 

```js
const empty = [];

const stillEmpty = [...empty];

console.log(stillEmpty); // []
```